src/modules/internalInterfaces.ts

```diff
@@ -2,3 +2,17 @@ export enum PublishingModuleType {
   CHANNEL = 'CHANNEL',
   THREAD = 'THREAD',
 }
+
+// NOTE: To maintain compatibility with the sendbirdSelector provided to customers
+//  this utility function ensures that if publishingModules is not provided or its length is zero
+//  it will maintain the same behavior as before.
+const publishTo = (target: PublishingModuleType, modules?: PublishingModuleType[]) => {
```

Comment:
nit; this sounds like an action handler which helps to publish modules to somewhere whereas it determines whether the target modules are published or not. 
How about `publishedTo` ?(not sure this what you meant 😅)

Reply:
Namespace is somewhat confusing.

change to something more explicit:
- isModuleTypeIncludedInModules

Reply:
ㅋㅋㅋ I wrote it without thinking properly. I'll make the necessary corrections.

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/842